server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -349,22 +390,43 @@ public QualityGateStatus getQualityGateStatus() {
 
   /**
    * Any Measure, which ever is its value type, can have a Variations.
+   * @deprecated as only one period is now available. Use {@link #hasVariation()} instead
    */
+  @Deprecated
   public boolean hasVariations() {
     return variations != null;
   }
 
   /**
+   * Any Measure, which ever is its value type, can have a variation.
+   */
+  public boolean hasVariation() {
+    return hasVariations() && variations.hasVariation1();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "hasVariations"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)